images.go

```diff
@@ -8,11 +8,11 @@ import (
 
 // ImageDTO represents a Shopify Image in HTTP requests and responses
 type ImageDTO struct {
-	CreatedAt time.Time `json:"created_at"`
-	SRC       string    `json:"src"`
-	Width     int       `json:"width"`
-	Height    int       `json:"height"`
-	Alt       string    `json:"alt"`
+	CreatedAt time.Time `json:"created_at,omitempty"`
+	SRC       string    `json:"src,omitempty"`
+	Width     int       `json:"width,omitempty"`
+	Height    int       `json:"height,omitempty"`
+	Alt       string    `json:"alt,omitempty"`
```

Comment:
I feel like mandatory fields shouldnt have this omit empty flag on right? Have you checked which fields are mandatory?